terraform/environments/mlra/lz-conn-test.tf

```diff
@@ -0,0 +1,46 @@
+locals {
```

Comment:
Do we want to commit this file to the main branch? Looks like it's from Mike's connectivity testing work

Reply:
Yes I believe this is still needed. Mike will be doing more connectivity tests back to databases etc in LAA LZ.